frontend/packages/core/src/AppLayout/stories/notifications.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import type { Meta } from "@storybook/react";
+
+import Notifications from "../notifications";
+
+export default {
+  title: "Core/AppLayout/Notifications",
+  component: Notifications,
+  parameters: {
+    backgrounds: {
+      default: "clutch",
+      values: [{ name: "clutch", value: "#131C5F" }],
```

Comment:
```suggestion
      values: [{ name: "header blue", value: "#131C5F" }],
```

Reply:
updated, addressed it on user story as well